CONTRIBUTING.md

```diff
@@ -1,70 +1,60 @@
 Contributing to nimi-python
 ===========================
 
-Contributions to **[nimi-python](https://github.com/ni/nimi-python)** are welcome from all!
+Contributions to **[`nimi-python`](https://github.com/ni/nimi-python)** are welcome from all!
 
-**[nimi-python](https://github.com/ni/nimi-python)** is managed via [Git](https://git-scm.com), with the canonical
+**[`nimi-python`](https://github.com/ni/nimi-python)** is managed via [Git](https://git-scm.com), with the canonical
 upstream repository hosted on [GitHub](http://developercertificate.org/).
 
-**[nimi-python](https://github.com/ni/nimi-python)** follows a pull request model for development.
+**[`nimi-python`](https://github.com/ni/nimi-python)** follows a pull request model for development.
 If you wish to contribute, you will need to create a GitHub account, fork this project,
 push a branch with your changes to your project, and then submit a pull request.
 
 See [GitHub's official documentation](https://help.github.com/articles/using-pull-requests/)
 for more details.
 
-Getting Started
----------------
 
-To contribute to this project, it is recommended that you follow these steps:
+Setting up your system
+----------------------
 
-### System Requirements
+In order to have the ability to build and run the tests you will need a few things to be set up on your system.
 
-In order to build **[nimi-python](https://github.com/ni/nimi-python)**, you must have the
-following installed:
+### Windows:
 
-* [Python](https://www.python.org/downloads/)
-    - If you're on Windows 10 (not [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about))
-        - Install the 64-bit Python version that is needed by `build_test` environment in [tox.ini](https://github.com/ni/nimi-python/blob/e13087eb67e2399de2dfa83fd504f8ebd0e0e263/tox.ini#L10)
-        - [Ensure](https://packaging.python.org/en/latest/tutorials/installing-packages/#ensure-you-can-run-pip-from-the-command-line) pip support is installed. 
+-  Install and enable [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide)
+- Install GNU Make: ``sudo apt-get install make``
+- Install zip: ``sudo apt-get install zip``
+- Install 64-bit [Python 3.9](https://www.python.org/downloads/)
```

Comment:
This will become stale when we add Python 3.10 support. Can we keep what we had before?
```
Install the 64-bit Python version that is needed by build_test environment in [tox.ini](https://github.com/ni/nimi-python/blob/e13087eb67e2399de2dfa83fd504f8ebd0e0e263/tox.ini#L10)
```